libs/ui-components/src/components/common/ButtonWithPermissions.tsx

```diff
@@ -0,0 +1,13 @@
+import { Button, ButtonProps } from '@patternfly/react-core';
+import * as React from 'react';
+
+type WithPermissions = {
+  permissions: [boolean, boolean, string | undefined];
```

Comment:
I don't understand this type well. The component is only using the first element?

Reply:
For now, yes. I'm keeping this open in case in other UIs (like AAP), we will want to disable + maybe show some tooltip.

I could remove it, if you prefer

Reply:
what's the third parameter (second boolean) for?
We could keep it, but maybe add some description?

Reply:
This is the result of `useAccessReview`. I've exported the type and used it here.

Reply:
Thanks for the clarification!